src/main/java/org/ccci/obiee/client/rowmap/impl/AnalyticsManagerFactoryImpl.java

```diff
@@ -106,6 +110,10 @@ public AnalyticsManager createAnalyticsManager()
                 converterStore,
                 tracer
             );
+        } catch (Exception e){
```

Comment:
Nitpicky, but maybe put a space before the `{`

Reply:
I don't mind nitpicks